client/src/helpers.ts

```diff
@@ -95,3 +97,14 @@ export function onTextInputChanged(
   });
   update();
 }
+
+export function loadSentry(): void {
+  Sentry.init({
+    dsn: "https://d137124c5ac546639e2536f860a92798@o1081855.ingest.sentry.io/6419221",
+    integrations: [new BrowserTracing()],
+
+    // We recommend adjusting this value in production, or using tracesSampler
+    // for finer control
+    tracesSampleRate: 0.1,
```

Comment:
how about 0 for v2 and say 0.2 for careerfair?

Reply:
Well that would incur the previous issue with gatsby not knowing which stage it's on, the same reason we put the lambda endpoints in the config. So I guess it would be helpful to also put a generic "server" field in the DB config that could be "v2" or "cf" for use cases such as this one?

Reply:
not anymore, the new cicd pipeline builds separately for each stage and can pass the stage parameter

Reply:
how does this look: [e1bc691](https://github.com/mentorpal/mentor-admin/pull/190/commits/e1bc691abd5543ad375a3ed414f14b36df13dc91)